js/components/cohort-definitions/cohort-definition-manager.js

```diff
@@ -574,6 +583,22 @@ define(['knockout', 'text!./cohort-definition-manager.html',
 			self.closeConceptSet();
 		}
 
+		self.showSaveConceptSet = function() {
+		  self.newConceptSetName(self.model.currentConceptSet().name());
+		  self.saveConceptSetShow(true);
+			// $("#saveConceptSetModal").modal('show');
+		};
+
+		self.hideSaveConceptSet = function() {
+		  self.saveConceptSetShow(false);
+			// $("#saveConceptSetModal").modal('hide');
+		};
+
+		self.saveConceptSet = function () {
+      console.log('save concept set', self.newConceptSetName());
```

Comment:
Could you remove debug logging and commented lines above if they are not required?